pkg/config/signer.go

```diff
@@ -18,6 +18,7 @@ import (
 	"io"
 	"math/big"
 
+	"github.com/gogo/protobuf/proto"
```

Comment:
Looks like your IDE auto-imported the wrong proto package. (Dereck and I hit this as well in our PR. :) )

Reply:
fixed 🔧